src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -602,13 +603,18 @@ private async Task ValidateViewModel(EditTenantViewModel model, bool newTenant)
                 ModelState.AddModelError(nameof(EditTenantViewModel.Name), S["The tenant name is mandatory."]);
             }
 
-            if (!String.IsNullOrWhiteSpace(model.FeatureProfile))
+            if (model.FeatureProfiles != null && model.FeatureProfiles.Any())
             {
                 var featureProfiles = await _featureProfilesService.GetFeatureProfilesAsync();
-                if (!featureProfiles.ContainsKey(model.FeatureProfile))
+
+                foreach (var featureProfile in model.FeatureProfiles)
                 {
-                    ModelState.AddModelError(nameof(EditTenantViewModel.FeatureProfile), S["The feature profile does not exist.", model.FeatureProfile]);
+                    if (!featureProfiles.ContainsKey(featureProfile))
+                    {
+                        ModelState.AddModelError(nameof(EditTenantViewModel.FeatureProfiles), S["The feature profile does not exist.", model.FeatureProfiles]);
```

Comment:
Did you mean to add `{0}` somewhere in the string?

Reply:
yes. I'll take care of it.